### ISSUE TITLE
Add downloading of "likes"

### ISSUE THREAD

**Issue by Shaav** [Liru/tumblr-downloader#45]
I came across this project because I was looking for a tool to download images/videos that I have "liked" using my own account. It would be *perfect* for my purposes if there was just a little flag somewhere that added the option to download from <username>.tumblr.com/likes rather than the blog itself... Seems like that wouldn't be a difficult thing to add?

**Comment by Shaav** [comment]
Yeah, so maybe not so simple. I thought maybe it would be easy enough I could just do it myself! Foolishly, I thought you were scraping the actual page --- should have known better there: who does that these days?? --- and could just add "likes" to the url. Of course, it's the api and there is no *.tumblr.com/likes/api/read/json equivalent. You need to use v2 of the api to get access to even publicly accessible likes which means registering an app to get an api key. That I have no problem with---did in fact---but even then it looks like the parameters for iterating calls through the post history are different. Probably the json is all different too. :P  Became too much for me right at this moment, especially since I've never used Go before.

So, anyway, I'd still love to see the functionality, but I understand that it probably only makes sense if you move to v2 of the api.

**Comment by Liru** [comment]
I'm still thinking about moving to the v2 API, but the original point of *this* project was to quickly download just users' images. If this is going to be added, it'll probably be added in v2.0 of this program, since I was planning on making some config files that would let you specify what should be downloaded from a certain user.

Making a separate program to download someone's "liked" posts should be easy enough once you get the API key, though.
